padevents/padevents.py

```diff
@@ -852,26 +852,30 @@ def make_full_guerrilla_output(self, table_name, event_list, starter_guerilla=Fa
         return header + tbl.get_string() + "\n"
 
     @commands.command(aliases=['events'])
-    async def eventsna(self, ctx):
+    async def eventsna(self, ctx, group=None):
         """Display upcoming daily events for NA."""
-        await self.do_partial(ctx, 'NA')
+        await self.do_partial(ctx, 'NA', group)
 
     @commands.command()
-    async def eventsjp(self, ctx):
+    async def eventsjp(self, ctx, group=None):
         """Display upcoming daily events for JP."""
-        await self.do_partial(ctx, 'JP')
+        await self.do_partial(ctx, 'JP', group)
 
     @commands.command()
-    async def eventskr(self, ctx):
+    async def eventskr(self, ctx, group=None):
         """Display upcoming daily events for KR."""
-        await self.do_partial(ctx, 'KR')
+        await self.do_partial(ctx, 'KR', group)
 
-    async def do_partial(self, ctx, server):
+    async def do_partial(self, ctx, server, group=None):
         server = normalize_server_name(server)
         if server not in SUPPORTED_SERVERS:
             await ctx.send("Unsupported server, pick one of NA, KR, JP")
             return
 
+        if group != None and group.lower() not in GROUPS:
```

Comment:
You're never supposed to compare singletons (like None) with the != operator.  This is just a wild python rule, and it only really every applies to None.  The way you're supposed to do this is `group is None` or the opposite `group is not None`.